extensions/sql-database-projects/src/test/project.test.ts

```diff
@@ -82,3 +83,23 @@ describe('Project: sqlproj content operations', function (): void {
 		await testUtils.shouldThrowSpecificError(async () => await project.addToProject(list), `ENOENT: no such file or directory, stat \'${nonexistentFile}\'`);
 	});
 });
+
+describe('Project: round trip updates', function (): void {
+	before(async function () : Promise<void> {
+		await baselines.loadBaselines();
+	});
+
+	it('Should update SSDT project to work in ADS', async function (): Promise<void> {
+		projFilePath = await testUtils.createTestSqlProjFile(baselines.SSDTProjectFileBaseline);
+		const project: Project = new Project(projFilePath);
+		await project.readProjFile();
+
+		await project.updateProjectForRoundTrip();
+
+		should(await exists(projFilePath+'_backup')).equal(true);	// backup file should be generated before the project is updated
```

Comment:
linter doesn't require spaces around the `+`?

Reply:
That didn't throw any error, but I will add spaces since you mentioned :-).